src/main/scala/com/gu/iosdeployments/Lambda.scala

```diff
@@ -15,7 +20,43 @@ object Lambda {
   }
 
   def process(env: Env): Unit = {
-    logger.info("Hello world")
-  }
+    logger.info("Loading configuration...")
+    val appStoreConnectConfig = AppStoreConnectConfig(env)
+    val appStoreConnectToken = JwtTokenBuilder.buildToken(appStoreConnectConfig)
+    val gitHubConfig = GitHubConfig(env)
+    logger.info("Successfully loaded configuration...")
+    val result = for {
+      runningDeployments <- GitHubApi.getRunningDeployments(gitHubConfig)
+      appStoreConnectBetaBuilds <- AppStoreConnectApi.getLatestBetaBuilds(appStoreConnectToken, appStoreConnectConfig)
+    } yield {
+      // Process one running deployment per lambda execution
+      // In practice it's extremely unlikely that there will be two concurrent deployments anyway
+      runningDeployments.headOption match {
+        case Some(runningDeployment) =>
+          val attemptToFindBeta = appStoreConnectBetaBuilds.find(_.version == runningDeployment.version)
+          (runningDeployment.environment, attemptToFindBeta) match {
+            case ("internal-beta", Some(LiveAppBeta(_, _, "IN_BETA_TESTING", _))) =>
+              logger.info(s"Internal beta deployment for ${runningDeployment.version} is complete...")
+            // GitHubApi.markDeploymentAsSuccess(gitHubConfig, runningDeployment.databaseId)
```

Comment:
Once we start marking deploys as complete there'll be a nice little audit trail of all deployments [here](https://github.com/guardian/ios-live/deployments).

Reply:
nice!